tests/crypto/ec_tests.c

```diff
@@ -808,40 +808,44 @@ static void _test_cert_extensions(
         {
             const Extension* ext = &extensions[i];
             const char* oid = ext->oid;
-            uint8_t data[4096];
-            size_t size = sizeof(data);
+            uint8_t* data = NULL;
+            size_t size = 0;
 
-            OE_TEST(oe_cert_find_extension(&cert, oid, data, &size) == OE_OK);
+            OE_TEST(oe_cert_find_extension(&cert, oid, &data, &size) == OE_OK);
             OE_TEST(strcmp(oid, ext->oid) == 0);
             OE_TEST(size == ext->size);
             OE_TEST(memcmp(data, ext->data, size) == 0);
+
+            free(data);
         }
     }
 
     /* Check for an unknown OID */
     if (!extensions)
     {
         oe_result_t r;
-        uint8_t data[4096];
-        size_t size = sizeof(data);
+        uint8_t* data = NULL;
+        size_t size = 0;
 
-        r = oe_cert_find_extension(&cert, "1.2.3.4", data, &size);
+        r = oe_cert_find_extension(&cert, "1.2.3.4", &data, &size);
         OE_TEST(r == OE_NOT_FOUND);
```

Comment:
recommend also adding:
```
OE_TEST(data == NULL);
```
to ensure it didn't return something that needs to be freed in this case